azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/service/sdk/wrapper/ChatEventTypeWrapper.kt

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.ui.chat.service.sdk.wrapper
+
+internal enum class ChatEventType {
+    CHAT_MESSAGE_RECEIVED,
+    CHAT_MESSAGE_EDITED,
+    CHAT_MESSAGE_DELETED,
+    TYPING_INDICATOR_RECEIVED,
+    READ_RECEIPT_RECEIVED,
+    CHAT_THREAD_CREATED,
+    CHAT_THREAD_DELETED,
+    CHAT_THREAD_PROPERTIES_UPDATED,
+    PARTICIPANTS_ADDED,
+    PARTICIPANTS_REMOVED,
+}
+
+internal fun ChatEventType.into(): com.azure.android.communication.chat.models.ChatEventType {
+    return when (this) {
+        ChatEventType.CHAT_MESSAGE_RECEIVED -> com.azure.android.communication.chat.models.ChatEventType.CHAT_MESSAGE_RECEIVED
```

Comment:
Is fully qualified name here necessary? Can maybe use Kotlin import alias?

Reply:
We have two types, one native SDK and one chat UI SDK, thus qualifiers help here